apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -209,6 +214,22 @@ body{
     border: none !important;
 }
 
+.pt-0 {
```

Comment:
these classes for padding-0 (pt-0, pb-0, pl-0, pr-0) is already defined in bootstrap.css, if we will add these here also then it will be overwrited. @prateek-giddh

Reply:
Done